pytrip/cube.py

```diff
@@ -431,8 +431,9 @@ def create_dicom_base(self):
         meta.TransferSyntaxUID = UID.ImplicitVRLittleEndian  # Implicit VR Little Endian - Default Transfer Syntax
         ds = FileDataset("file", {}, file_meta=meta, preamble=b"\0" * 128)
         ds.PatientsName = self.patient_name
-        if self.patient_id is None:
-            ds.PatientID = datetime.datetime.today().strftime('%Y%m%d-%H%M%S')
+        if self.patient_id is None or self.patient_id == '':
+            self.patient_id = datetime.datetime.today().strftime('%Y%m%d-%H%M%S')
+        ds.PatientID = self.patient_id
```

Comment:
This may introduce some bugs in the future, as before the change `create_dicom_base` method wasn't modifying self object and now it is (in case patient_id is missing).

Fixing missing patient_id shouldn't be done here.